src/ofxJSONElement.h

```diff
@@ -13,7 +13,12 @@
 #include <iostream>
 #include <fstream>
 #include <json/json.h>
-#include "ofMain.h"
+
+// IOSP-40 // #include "ofMain.h"
+
+#include <vector>
+
+using namespace std;
```

Comment:
Generally we try to avoid `using namespace std` (although it is used in the of core, we are in the process of removing it.